Description: A patch for main.c before 0.31.1-16
 This patch has been created to represent all changes was made direct
 to the package before implement dpkg-src 3.0(quilt) format, which has 
 a patch system.
 .
 Just a cast and force the use of int for 32 and 64 bits.
Author: Edgar Antonio Palma de la Cruz <xbytemx@gmail.com>
Last-Update: 2011-08-07

--- autotrace-0.31.1.orig/main.c
+++ autotrace-0.31.1/main.c
@@ -34,16 +34,16 @@ static char * output_name = (char *)"";
 static at_output_write_func output_writer = NULL;
 
 /* Whether to print version information */
-static at_bool printed_version;
+static int printed_version;
 
 /* Whether to write a log file */
-static at_bool logging = false;
+static int logging = false;
 
 /* Whether to dump a bitmap file */
-static at_bool dumping_bitmap = false;
+static int dumping_bitmap = false;
 
 /* Report tracing status in real time (--report-progress) */
-static at_bool report_progress = false;
+static int report_progress = false;
 #define dot_printer_max_column 50
 #define dot_printer_char '|'
 static void dot_printer(at_real percentage, at_address client_data);
@@ -322,7 +322,7 @@ read_command_line (int argc, char * argv
 	    str = "big";
 	  
 	  printf("%d bit, %s endian\n",
-		 sizeof(void *) * 8,
+		 (int)(sizeof(void *) * 8),
 		 str);
 	  exit(0);
 	}
